src/kemal/config.cr

```diff
@@ -24,6 +24,8 @@ module Kemal
       @error_handler = nil
       @always_rescue = true
       @server = uninitialized HTTP::Server
+      @dir_listing = false
+      @enable_gzip = false
```

Comment:
That makes sense. Let's make `gzip = true` and `dir_listing = false` by defualt.

The remaining question is how to  totally disable `serve_static`.